stdlib/public/Windows/WinSDK.swift

```diff
@@ -93,3 +93,64 @@ public extension FILETIME {
   }
 }
 
+// WindowsBOOL
```

Comment:
Hm. We camel-cased ObjCBool even though the original is BOOL there too. On the other hand, Windows programmers are used to seeing all caps in low-level type names…

Reply:
I really was tempted to create `BOOL` instead so that you could directly use the `BOOL` type still.  Honestly, I would prefer if `WindowsBOOL` was an implementation detail that never was seen by users - with the two options being use `BOOL` or `Bool`.

Reply:
The place where it comes up is if someone has a pointer to it. If you think `BOOL` is the right name in that context, you can use it…but we thought that would be too confusing when `Bool` is the thing you use normally. It's useful to be able to talk about the difference out loud, for example.

Reply:
So, my concern was in the same vein, but, was more concerned about people getting confused over why `BOOL` is available on Windows and maps properly but is not available on Darwin.  The really nice thing about using `BOOL` though is that it is the same spelling as the underlying type so you can really easily map it back.

Reply:
I mean, that's true for ObjC's `BOOL` or Darwin's `Boolean`, but we don't actually *want* people to use those, and similarly we don't want people writing `BOOL` for Windows APIs either in the common case.

Reply:
Agreed, in the common case, we really want people to use the `Bool` type.  I'm OK with leaving this as `WindowsBOOL` or changing it to `WindowsBool`.  I honestly don't have a strong opinion on this - particularly if we are venturing away from `BOOL`.  So, if you have a preference, I'm happy to go with that.

Reply:
I guess I have a mild preference for `WindowsBool` over `WindowsBOOL` for consistency with Apple platforms, even though few people will see that consistency.

Reply:
Works for me.  I can see the consistency in that.